tests/fixtures/tasks/sample_task.json

```diff
@@ -0,0 +1,124 @@
+{
```

Comment:
Is it different from the one above? Can we re-use?

Reply:
This one is associated with an app, the other is not. To me, it made sense to have both examples in the test.

Reply:
Just asking. :)